pipelines/datasets/br_sfb_sicar/flows.py

```
@@ -24,96 +48,241 @@
+    back to AllFree.
+    """
+    from google.cloud import bigquery
+
+    client = bigquery.Client(project=billing_project)
+    sql = (
+        f"SELECT MIN(data) AS mn, MAX(data) AS mx "
+        f"FROM `basedosdados.{DATASET_ID}.{table_id}`"
+    )
+    try:
+        row = next(iter(client.query(sql).result()))
+    except Exception as exc:  # table missing on the very first prod run
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)